src/content/docs/accounts/accounts/saml-single-sign/saml-service-providers.mdx

```diff
@@ -55,20 +55,21 @@ If you don't see this UI, review the [requirements](#requirements).
 
 ## Providers supported by New Relic [#saml_providers]
 
-For a list of the SAML service providers that New Relic currently supports for SSO integration: From the New Relic title bar, select **([account dropdown](/docs/accounts-partnerships/education/getting-started-new-relic/glossary#account-dropdown)) > Account settings > Security and authentication > Single sign-on**.
+For a list of the SAML service providers that New Relic currently supports for SSO integration for users on our [original user model](/docs/accounts/original-accounts-billing/original-users-roles/overview-user-models): From the New Relic title bar, select **([account dropdown](/docs/accounts-partnerships/education/getting-started-new-relic/glossary#account-dropdown)) > Account settings > Security and authentication > Single sign-on**. **If you don't see that UI, it may be because you're on the [New Relic One user model](/docs/accounts/original-accounts-billing/original-users-roles/overview-user-models)**: in that case, you'll use [a different method to set up SAML SSO](/docs/accounts/accounts-billing/new-relic-one-user-management/introduction-saml-scim).
```

Comment:
```suggestion
Users on our [original user model](/docs/accounts/original-accounts-billing/original-users-roles/overview-user-models) can find a list of the SAML service providers that New Relic currently supports for SSO integration: From the New Relic title bar, select **([account dropdown](/docs/accounts-partnerships/education/getting-started-new-relic/glossary#account-dropdown)) > Account settings > Security and authentication > Single sign-on**. **If you don't see that UI, it may be because you're on the [New Relic One user model](/docs/accounts/original-accounts-billing/original-users-roles/overview-user-models)**: in that case, you'll use [a different method to set up SAML SSO](/docs/accounts/accounts-billing/new-relic-one-user-management/introduction-saml-scim).
```